pkg/jsonSchemaValidator/validator.go

```diff
@@ -70,12 +74,21 @@ func (jsv *JSONSchemaValidator) Validate(schemaContent string, yamlContent []byt
 	compiler.RegisterExtension("customKeyRule101", extensions.CustomKeyRule101, extensions.CustomKeyRule101Compiler{})
 	compiler.RegisterExtension("customKeyRegoRule", extensions.CustomKeyRegoRule, extensions.CustomKeyRegoDefinitionCompiler{})
 
-	schema, err := compiler.Compile("schema.json")
-	if err != nil {
-		return nil, err
+	// compiler.Compile() is an expensive operation. We cache the compiled schema in rulesSchemasCache to avoid re-compiling the same schema.
+	if _, ok := jsv.rulesSchemasCache.Load(schemaContent); !ok {
```

Comment:
I think it is better calling the `ok` variable something like `schemaContentExists` or anything similar

Reply:
meaningful name vs standard name
both options are legit, @TzlilSwimmer123 and I prefer the standard name in this case (`ok`)